tedpermission/src/main/res/values-v11/styles.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+
+  <style name="Theme.Transparent.Permission"
```

Comment:
First of all, make base theme `Theme.Transparent.Permission.Base` in `values/styles.xml`.
And you can extends that style each `values-xx/style.xml`
- values/styles.xml
```java
 <style name="Theme.Transparent.Permission.Base"
      parent="Theme.AppCompat.Light.NoActionBar">
    <item name="android:windowIsTranslucent">true</item>
    <item name="android:windowBackground">@android:color/transparent</item>
    <item name="android:windowContentOverlay">@null</item>
    <item name="android:windowIsFloating">true</item>
    <item name="android:backgroundDimEnabled">false</item>

    <item name="android:windowActionBar">false</item>
    <item name="android:windowNoTitle">true</item>
    <item name="windowNoTitle">true</item>
  </style>

<style name="Theme.Transparent.Permission"
      parent="Theme.Transparent.Permission.Base">
</style>
```
- values-v11/styles.xml
```java
<style name="Theme.Transparent.Permission"
      parent="Theme.Transparent.Permission.Base">
    <item name="android:windowActionBar">false</item>
</style>
```

Reply:
I agree with you. It looks better!
I'll push changes!

Thanks.